.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -382,6 +382,45 @@ and Linux `tsh` binaries are tested for FIDO2 items.
   - [ ] `tsh touchid ls` works
   - [ ] `tsh touchid rm` works (careful, may lock you out!)
 
+### Hardware Key Login
+
+Hardware Key Login is an Enterprise feature and is not available for OSS.
+
+On Linux, hardware key support requires that `tsh` and `teleport` are both built with libpcsclite. Ask for a statically-built `tsh` and `teleport` binary for realistic tests.
+
+You will need a YubiKey 4.3+ for each test. `tsh logout` between each section.
+
+- [ ] `auth_service.authentication.require_session_mfa: hardware_key`
```

Comment:
I'd also add a test to enable hardware keys on an existing cluster. This should force you to re-login with YK. I'm not sure if this is something that we want to include in the test plan, but most of our existing users will have to go through that if they decide to enable this feature.
WDYT?

Reply:
Agreed, I didn't want to introduce sequentiality to the tests but I think I found a good flow for it now.